atomicapp/providers/openshift.py

```diff
@@ -461,7 +461,11 @@ def undeploy(self):
             if self.dryrun:
                 logger.info("DRY-RUN: DELETE %s", url)
             else:
-                self.oc.delete(url)
+                try:
+                    self.oc.delete(url)
+                except Exception as e:
+                    if not self.ignore_errors:
+                        raise e
```

Comment:
the self.oc.scale() above (line 459) will fail if the rc doesn't exist in the target. We may need to consider putting the try/except deeper in the code. One extreme would be to put it at the Utils.make_rest_request() level, the other extreme would be to simply try/except around the scale call above. Thoughts?